.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
```

Comment:
This is a nice catch, as apparently, the default is `true` 👍

Reply:
Yes I start to have some good experience with Github actions, this caught me a few times! 😄